.travis.yml

```diff
@@ -43,7 +47,8 @@ install:
   - case "$BUILD" in
       stack)
         $STACK setup --no-terminal;
-        $STACK build --only-snapshot --no-terminal;;
+        $STACK build --only-snapshot --no-terminal;
+        $STACK build --no-terminal happy;;
```

Comment:
Does ;; have to be at the end of the last line? I'm looking for a way to format this such that I'd only see a single line diff, not 2 lines.